Makefile

```diff
@@ -163,6 +163,12 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 .PHONY: deploy
 deploy: kind deploy-cluster deploy-controller ## Deploy cluster and controller.
 
+.PHONY: deploy-hostmode
+deploy-hostmode: kind deploy-cluster ## Deploy cluster and controller in hostmode, then setup systemd services.
+	./systemdmode/setup_node_config.sh $(KIND_CLUSTER_NAME)
+	$(MAKE) deploy-controller KUSTOMIZE_LAYER=hostmode
+	./systemdmode/deploy.sh $(KIND_CLUSTER_NAME)
+
```

Comment:
You can play with target deps so you don't have to call $(MAKE) again.
Something like the following, syntax or order could be wrong, I have just do it from memory:
```
.PHONY: setup-hostmode
setup-hostmode:
    ./systemdmode/setup_node_config.sh $(KIND_CLUSTER_NAME)
.PHONY: deploy-hostmode
deploy-hostmode: KUSTOMIZE_LAYER=hostname
deploy-hostmode: setup-hostmode deploy-controller
    ./systemdmode/deploy.sh $(KIND_CLUSTER_NAME)
```